pandas/core/apply.py

```diff
@@ -308,15 +313,6 @@ def wrap_results(self):
 class FrameRowApply(FrameApply):
     axis = 0
 
-    def get_result(self):
-
-        # dispatch to agg
-        if isinstance(self.f, (list, dict)):
-            return self.obj.aggregate(self.f, axis=self.axis,
-                                      *self.args, **self.kwds)
```

Comment:
``if isinstance(self.f, (list, dict))`` should also be called when axis=1, so moved up to ``FrameApply.get_results``.